modernization-definition.md

```diff
@@ -0,0 +1,21 @@
+# Defining Mainframe Modernization
+
+Businesses are digitally transforming to better serve their customers,
+reduce time to market, and increase their ability to respond quickly to
+market changes, consumer behavior, and new regulations. As businesses
+transform, they are imposing significant demands on existing mainframe
+applications and data, driving the need for modernization to better
+achieve business outcomes.
+
+It's often not a question of if you need to modernize mainframe
```

Comment:
Now that we're in the review cycle for this PR, it's appropriate to comment on the specific lines of this proposed text.

Reply:
Ok boss.